test_suite/package/test_otel.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+import logging
+import time
+import json
+from lib import console_lib
+from test_suite.generic.test_generic import TestsSubscriptionManager
+from lib import test_lib
+logger = logging.getLogger(__name__)
```

Comment:
Is there a reason why we are keeping this logger after the last review conversation?